packages/cli/src/index.ts

```diff
@@ -764,6 +777,66 @@ yargs
       commandWrapper(handlerBuildCommand)(argv as Arguments<BuildDocsArgv>);
     }
   )
+  .command(
+    'translations',
+    'Creates or updates translations.yaml files and fills them with missing built-in translations and translations from the redocly.yaml and sidebars.yaml files.',
+    (yargs) =>
+      yargs.options({
+        contentDir: {
```

Comment:
Could we use `source-dir` here, to be consistent with other commands such as `preview`

Reply:
Ah, I thought it was the directory where the project itself lived, so that the translations could be added to that structure as appropriate. Now I'm not clear if this argument is input or output! Can we get some clarification?

Reply:
it should still be kebab-case, whatever we call it! "Content" sounds like the content to translate, but I think this is actually the directory where the translations should live, is that right? Maybe `translations-dir` if that's the case